src/components/TodoForm/TodoForm.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { useState } from 'react';
+import usersFromServer from '../../api/users';
+import { User } from '../../types/User';
+import { Todo } from '../../types/Todo';
+import { getUserById } from '../../services/user';
+
+type Props = {
+  onSubmit: (todo: Todo) => void;
+};
+
+export const TodoForm: React.FC<Props> = ({ onSubmit }) => {
+  const [title, setTitle] = useState('');
+  const [hasTitleErrors, setHasTitleErrors] = useState(false);
+
+  const [userId, setUserId] = useState(0);
+  const [hasUserIdErrors, setHasUserIdErrors] = useState(false);
+
+  const handleTitleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
the prefix `handle` doesn't say anything about the function,
so the prefix is not required